### ISSUE TITLE
flaky test: replication-luatest/linearizable_test.lua

### ISSUE THREAD

**Issue by Gerold103** [tarantool/tarantool#7737]
Execute `python test-run.py $(yes linearizable | head -n 15)` and some of the runs will hang.

**Comment by Gerold103** [comment]
After the problem with `ER_READONLY` bootstrap is fixed in #7752, I get this error (when run many in parallel, like above):
```
[008] replication-luatest/linearizable_test.lua                       [ fail ]
[008] Test failed! Output from reject file /tmp/t/rejects/replication-luatest/linearizable.reject:
[008] wait vclock: --- {1: 3, 2: 3}
[008] ...
[008]  to ---
[008] - 3
[008] - 5
[008] ...
[008] 
[008] wait vclock: --- {1: 3, 2: 5}
[008] ...
[008]  to --- {1: 3, 2: 5, 3: 1}
[008] ...
[008] 
[008] wait vclock: --- {0: 1, 1: 3, 2: 5, 3: 3}
[008] ...
[008]  to --- {1: 3, 2: 5, 3: 4}
[008] ...
[008] 
[008] TAP version 13
[008] 1..6
[008] # Started on Fri Sep 30 23:20:24 2022
[008] # Starting group: linearizable-read
[008] ok     1	linearizable-read.test_wait_others
[008] ok     2	linearizable-read.test_timeout
[008] not ok 3	linearizable-read.test_no_dirty_reads
[008] #   ...tarantool/test/replication-luatest/linearizable_test.lua:162: Confirmed data is seen
[008] #   expected: {1}
[008] #   actual: nil
[008] #   stack traceback:
[008] #   	...tarantool/test/replication-luatest/linearizable_test.lua:152: in function 'linearizable-read.test_no_dirty_reads'
[008] #   	...
[008] #   	[C]: in function 'xpcall'
[008] ok     4	linearizable-read.test_reconnect_while_waiting
[008] ok     5	linearizable-read.test_leader_change
[008] # Starting group: linearizable-read-basic
[008] ok     6	linearizable-read-basic.test_basic
[008] # Ran 6 tests in 8.806 seconds, 5 succeeded, 1 failed
```

**Comment by sergepetrenko** [comment]
Duplicate of tarantool/tarantool-qa#277
